src/Components/NotificationIntegrationModal/NotificationIntegrationModal.jsx

```
@@ -0,0 +1,420 @@
+  Typography, 
+  Box,
+  Tabs,
+  Tab,
+  Divider
+} from "@mui/material";
+import { useTheme } from "@emotion/react";
+import TextInput from "../Inputs/TextInput";
+import Checkbox from "../Inputs/Checkbox";
+
+// Tab Panel component
+function TabPanel({ children, value, index, ...other }) {
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)